peerconnection.go

```diff
@@ -992,6 +992,29 @@ func (pc *PeerConnection) SetRemoteDescription(desc SessionDescription) error {
 			}
 
 			t, localTransceivers = findByMid(midValue, localTransceivers)
+			// Fix simulcast tracks removing
+			if t != nil && t.Receiver() != nil && len(t.Receiver().Tracks()) > 1 {
+				switch direction {
+				case RTPTransceiverDirectionInactive:
+					if err := t.Stop(); err != nil {
+						return err
+					}
+				case RTPTransceiverDirectionRecvonly:
+					if recv := t.Receiver(); recv != nil {
+						if err := recv.Stop(); err != nil {
+							return err
+						}
+					}
+				case RTPTransceiverDirectionSendonly, RTPTransceiverDirectionSendrecv:
```

Comment:
Do we need `RTPTransceiverDirectionSendrecv` here?

Reply:
In my tests, after the track is re attached in `sendrecv` transceiver it changes to `sendonly` so it may not be required just let it there as precaution, but it can be removed.